utils/src/test/java/de/hellfish/ngrok/utils/HttpRequestSerializerTest.java

```diff
@@ -0,0 +1,50 @@
+package de.hellfish.ngrok.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class HttpRequestSerializerTest {
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+        objectMapper = new ObjectMapper();
+    }
+
+    @Test
+    void testWriteToOutputStream() throws IOException {
```

Comment:
Well, the problem here that you completely mimic logic of the underlying  HttpRequestSerializer but in reality you shouldn't care how a object is packed to I/O streams.

Consider to rewrite this test in a way, where the first part converts an object to a stream using `HttpRequestSerializer` and the other part deserializes a stream to an object back (using`HttpRequestSerializer` as well).